pkg/agent/openflow/pipeline.go

```diff
@@ -347,11 +366,14 @@ var (
 
 	// snatPktMarkRange takes an 8-bit range of pkt_mark to store the ID of
 	// a SNAT IP. The bit range must match SNATIPMarkMask.
-	snatPktMarkRange = binding.Range{0, 7}
+	snatPktMarkRange    = binding.Range{0, 7}
+	servicePktMarkRange = binding.Range{8, 8}
```

Comment:
Please add comments to explain this mark. And I guess you should rename it to better reflect its use.

Reply:
servicePktMarkRange takes a 1-bit range of pkt_mark to mark the first packet of NodePort/LoadBalancer packet whose externalTrafficPolicy is Local and Endpoint is on host network. There is a iptables entry which is used to match the pkt_mark and add a ct_mark to the connection.